committee-code-of-conduct/governance/onboarding-offboarding.md

```diff
@@ -0,0 +1,43 @@
+# Onboarding and offboarding new Code of Conduct Committee members
+
+Different actions on this list must be carried out by different members: 
+
+- **Outgoing members:** members who are ending their term
+- **Incoming members:** members beginning their term
+- **Carryover members:** members mid-way through their term
+
+## Permissions 
+
+### Slack
+
+**Who executes:** When offboarding, outgoing members must remove themselves from Slack channels. When onboarding, carryover members must add incoming members.
+
+- [ ] Code of conduct committee Slack channel on `kubernetes.slack.com`
```

Comment:
Should this list specific #channel names for clarity?  We were added to two in the k8s slack.